data/templates/default.latex

```diff
@@ -156,6 +156,7 @@ $endif$
 $if(links-as-notes)$
 % Make links footnotes instead of hotlinks:
 \renewcommand{\href}[2]{#2\footnote{\url{#1}}}
+\DeclareRobustCommand{\href}[2]{#2\footnote{\url{#1}}}
```

Comment:
Oh, no. I just blindly copied the the line from the other changeset and forgot to remove the prev line. I will fix this ASAP.